Fix failing TestAccPacketDevice_importBasic

it fails in the TF TeamCity

```
------- Stdout: -------
=== RUN   TestAccPacketDevice_importBasic
--- FAIL: TestAccPacketDevice_importBasic (65.01s)
    testing.go:569: Step 1 error: ImportStateVerify attributes not equivalent. Difference is shown below. Top is actual, bottom is expected.
        
        (map[string]string) {
        }
        
        
        (map[string]string) (len=1) {
         (string) (len=32) "wait_for_reservation_deprovision": (string) (len=5) "false"
        }
        
FAIL
```

--- a/packet/resource_packet_device.go
+++ b/packet/resource_packet_device.go
@@ -525,6 +525,11 @@ func resourcePacketDeviceRead(d *schema.ResourceData, meta interface{}) error {
 	}
 	d.Set("network_type", device.NetworkType)
 
+	wfrd := "wait_for_reservation_deprovision"
+	if _, ok := d.GetOk(wfrd); !ok {
+		d.Set(wfrd, nil)
+	}
+
 	d.Set("tags", device.Tags)
 	keyIDs := []string{}
 	for _, k := range device.SSHKeys {

